packages/react-topology/src/components/TopologyView/examples/Topology.md

```diff
@@ -1,155 +1,71 @@
 ---
 id: Topology view
-section: components
+section: extensions
```

Comment:
Can you import any of the propComponents to build any relevant prop tables at the bottom of these docs? And if you add `cssPrefix: pf-topology` as a line here, it might add the CSS variables used by topology to the bottom of our docs to more closely mirror the other component docs

Reply:
I guess for propComponents, you could use all or most of the most important components you're importing from the topology package in your examples.

Reply:
It's not pulling in the CSS or the props correctly. Some of the components get some of their props but just a few.
I'd like to defer this to another update once we get it working correctly